packages/frontend/src/components/select_sports.tsx

```diff
@@ -0,0 +1,61 @@
+import { Sport } from "@esp-group-one/types";
+import { useState } from "react";
+import type { ChangeEvent } from "react";
+
+export function SelectSport() {
+  const [sport, setSport] = useState("Tennis");
+  const [colour, setColour] = useState("tennis");
+
+  const options = [
+    { colour: "tennis", label: "Tennis" },
+    { colour: "badminton", label: "Badminton" },
+    { colour: "squash", label: "Squash" },
+  ];
+
+  const handleChange = (e: ChangeEvent<HTMLSelectElement>) => {
+    setSport(e.target.value);
+    if (e.target.value === "Tennis") {
```

Comment:
This is really not very future proof!

You should be assigning the value to the `Sport` which you can then pass straight to `setColour`.

You will then have to do the inverse with the label. However as mentioned above you can simply uppercase the first letter. (which will also mean that you don't have to store two different states (instead just store the current `Sport` which is selected).

Finally, if doing if statements like this, use `switch`!